build.gradle

```diff
@@ -80,6 +80,12 @@ configure(javaProjects) {
         }
     }
 
+    test {
+        testLogging {
+            events "passed", "skipped", "failed"
```

Comment:
(Useful mostly on Travis CI, where a failure does not say anything about what test failed and there's no easy way to get the unittest reports)